src/main/scala/cc/factorie/app/nlp/load/XMLSectionalizer.scala

```diff
@@ -70,32 +72,44 @@ class XMLSectionalizer(boundaryToken:String, excludeTokens:Set[String]) extends
           }
           stateStack.pop()
           tokenBuffer += t
+//          println("token:"+t + "\t" + "accepted close tag" + "\tUsable" )
         case (excludedOpenTag(tag), Usable) =>
           if(tokenBuffer.nonEmpty) {
-            sectionBuffer += new UsableText(tokenBuffer)
+            // NOTE (Ajay, Melisa - 5/3/2016) : To filter out the following tags in text <P> , </P>, =(FOTOS)=
+            sectionBuffer += new UsableText(tokenBuffer.filter(x => !(x.string.contentEquals("<P>") ||
+                                                                      x.string.contentEquals("</P>") ||
+                                                                      x.string.contentEquals("=(FOTOS)=")) ))
             tokenBuffer.clear()
           }
           tokenBuffer += t
           stateStack push Unusable
+//          println("token:"+t + "\t" + "excluded open tag" + "\tUsable" )
         case (excludedOpenTag(tag), Unusable) =>
           tokenBuffer += t
           stateStack push Unusable
+//          println("token:"+t + "\t" + "excluded open tag" + "\tUnusable" )
         case (excludedCloseTag(tag), Unusable) if tagStack.headOption == Some(tag.asInstanceOf[String]) =>
           tagStack.pop()
           sectionBuffer += new UnusableText(tokenBuffer)
           tokenBuffer.clear()
           stateStack.pop()
+//          println("token:"+t + "\t" + "excluded close tag" + "\tUnusable" )
         case (acceptedCloseTag(tag), Unusable) =>
           // we are in this state because we found an excluded open tag without a corresponding close tag.
           // In that event we just read in everything as usable text
           if(tokenBuffer.nonEmpty) {
-            sectionBuffer += new UsableText(tokenBuffer)
+            // NOTE: (Ajay, Melisa - 5/3/2016) To filter out the following tags in text <P> , </P>, =(FOTOS)=
+            sectionBuffer += new UsableText(tokenBuffer.filter(x => !(x.string.contentEquals("<P>") ||

```

Comment:
Yes, and also use a list of tokens that we want to filter instead of multiple or clauses.